cumulusci/tasks/release_notes/task.py

```diff
@@ -13,6 +13,39 @@
 )
 
 
+class AllGithubReleaseNotes(BaseGithubTask):
+
+    task_options = {
+        "repos": {
+            "description": (
+                "The list of owner, repo key pairs to generate release notes for." + " Ex: 'owner': SalesforceFoundation \
+                                                                                           'repo': 'NPSP'"
+            ),
+            "required": False,
+        },
+
+    }
+
+    def _run_task(self):
+        table_of_contents = "<h1>Table Of Contents</h1><ul>"
+        body = ""
+        for project in self.options["repos"]:
+            print(f"Owner: {project['owner']}")
+            print(f"Repo: {project['repo']}")
+            if project["owner"] and project["repo"]:
+                release = self.github.repository(project["owner"],project["repo"]).latest_release().body
+                table_of_contents += f"<a href=#{project['repo']}><li><slot name={project['repo']} />{project['repo']}</li></a>"
+                release_project_header = f"<h1 slot id={project['repo']} name={project['repo']}>{project['repo']}</h1><br>"
+                release_html = self.github.markdown(release,mode="gfm",context="{}/{}".format(project["owner"],project["repo"]))
+                body += f"{release_project_header}{release_html}"
+        table_of_contents += "</ul>"
+        
+        result = table_of_contents + body
+        result = result.replace("\n","<br>") # fixing newline issue
```

Comment:
Say more - what's the newline issue? Can it be corrected closer to its source? It seems to be constraining you to construct your HTML without any line breaks to avoid extra `<br>` tags, which is a little clunky.

Reply:
So I `suspect` that its reading newline raw bytes and as a result an extra \n is being displayed in the html display. I replaced it with a <br> to `html`ify it.